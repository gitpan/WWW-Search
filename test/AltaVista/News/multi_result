http://www.altavista.digital.com/cgi-bin/query?what=news&text=yes&fmt=d&pg=q&q=perl
/cgi-bin/query?pg=q&what=news&text=yes&q=perl&stq=30
/cgi-bin/query?pg=q&what=news&text=yes&q=perl&stq=60
/cgi-bin/query?pg=q&what=news&text=yes&q=perl&stq=90
/cgi-bin/query?pg=q&what=news&text=yes&q=perl&stq=120
/cgi-bin/query?pg=q&what=news&text=yes&q=perl&stq=150
/cgi-bin/query?pg=q&what=news&text=yes&q=perl&stq=180
