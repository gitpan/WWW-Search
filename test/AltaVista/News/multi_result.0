<html><head>
<title>AltaVista: Simple Query +Perl +CPAN</title>
</head><body>
<font size=+1 face=arial, helvetica>
<a href="/cgi-bin/query?text">AltaVista Search Service</a>
</font>
<br>
<FORM name=mfrm method=GET action="/cgi-bin/query">
<INPUT TYPE=hidden NAME=pg VALUE=q>
<INPUT TYPE=hidden NAME=what VALUE="news">
<INPUT TYPE=hidden NAME=text VALUE=yes>
<font size=-1><b>Search Usenet for postings on:</b>
</font><br><INPUT NAME=q size=50 maxlength=800 VALUE="+Perl +CPAN">
 <input type=submit name=search value="Search"><br><P><font size=-1>Search - <a href="/cgi-bin/query?pg=aq&what=news&text=yes">Advanced</a> - <a href="/cgi-bin/query?pg=q&text=yes">The Web</a></font><BR></FORM>
<font size=-1><b>AltaVista found about 2539 postings for you. </font></b><P><font size=-1><b>Result Pages: </b><b>1</b> &nbsp;<a href="/cgi-bin/query?pg=q&what=news&text=yes&q=%2bPerl+%2bCPAN&stq=30&c9k">2</a>
 &nbsp;<a href="/cgi-bin/query?pg=q&what=news&text=yes&q=%2bPerl+%2bCPAN&stq=60&c9k">3</a>
 &nbsp;<a href="/cgi-bin/query?pg=q&what=news&text=yes&q=%2bPerl+%2bCPAN&stq=90&c9k">4</a>
 &nbsp;<a href="/cgi-bin/query?pg=q&what=news&text=yes&q=%2bPerl+%2bCPAN&stq=120&c9k">5</a>
 &nbsp;<a href="/cgi-bin/query?pg=q&what=news&text=yes&q=%2bPerl+%2bCPAN&stq=150&c9k">6</a>
 &nbsp;<a href="/cgi-bin/query?pg=q&what=news&text=yes&q=%2bPerl+%2bCPAN&stq=180&c9k">7</a>
&nbsp; <a href="/cgi-bin/query?pg=q&what=news&text=yes&q=%2bPerl+%2bCPAN&stq=30&c9k">[<b>Next &gt;&gt;</b>]</a> <br>
</font><dl>
<dt><b>1. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@7114@de%2ecomp%2elang%2eperl"><strong>GD-Modul aus der CPAN in Perl einbinden</strong></a><dd>15 Sep 98 - <b>de.comp.lang.perl</b><br><a href="news:35fe1c41.0@isoit370.bbn.hp.com">&lt;35fe1c41.0@isoit370.bbn.hp.com&gt;</a><br>  <a href="mailto:RaymondBurks@Non-HP-Germany-om88.om.hp.com">&quot;raymond Burks&quot; &lt;Raymond Burks@Non-HP-Germany-om88.om.hp.com&gt;</a><P>
<dt><b>2. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@15693@comp%2elang%2eperl%2emodules"><strong>Does the CPAN module work with win32 Perl??</strong></a><dd>10 Nov 98 - <b>comp.lang.perl.modules</b><br><a href="news:36489D5C.D09B75A9@nist.gov">&lt;36489D5C.D09B75A9@nist.gov&gt;</a><br>  <a href="mailto:glenn.forney@nist.gov">&quot;Glenn P. Forney&quot; &lt;glenn.forney@nist.gov&gt;</a><P>
<dt><b>3. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@15696@comp%2elang%2eperl%2emodules"><strong>Re: Does the CPAN module work with win32 Perl??</strong></a><dd>10 Nov 98 - <b>comp.lang.perl.modules</b><br><a href="news:slrn74habf.i2r.randy@theory.uwinnipeg.ca">&lt;slrn74habf.i2r.randy@theory.uwinnipeg.ca&gt;</a><br>  <a href="mailto:randy@theory.uwinnipeg.ca">randy@theory.uwinnipeg.ca (Randy Kobes)</a><P>
<dt><b>4. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@15716@comp%2elang%2eperl%2emodules"><strong>Re: Does the CPAN module work with win32 Perl??</strong></a><dd>11 Nov 98 - <b>comp.lang.perl.modules</b><br><a href="news:36495AD2.9C29C9D6@ndirect.co.uk_NOSPAM">&lt;36495AD2.9C29C9D6@ndirect.co.uk_NOSPAM&gt;</a><br>  <a href="mailto:msergeant@ndirect.co.uk_NOSPAM">Matt Sergeant &lt;msergeant@ndirect.co.uk_NOSPAM&gt;</a><P>
<dt><b>5. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@124375@comp%2elang%2eperl%2emisc"><strong>ANNOUNCE: Data::Locations 4.1</strong></a><dd> 6 Jun 98 - <b>comp.lang.perl.misc</b><br><a href="news:6lb0oo$5p4$1@en1.engelschall.com">&lt;6lb0oo$5p4$1@en1.engelschall.com&gt;</a><br>  <a href="mailto:sb@engelschall.com">sb@engelschall.com (Steffen Beyer)</a><P>
<dt><b>6. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@124955@comp%2elang%2eperl%2emisc"><strong>ANNOUNCE: Date::Calc 4.1</strong></a><dd> 9 Jun 98 - <b>comp.lang.perl.misc</b><br><a href="news:6lj68i$gs3$3@bsdti6.sdm.de">&lt;6lj68i$gs3$3@bsdti6.sdm.de&gt;</a><br>  <a href="mailto:sb@sdm.de">Steffen Beyer &lt;sb@sdm.de&gt;</a><P>
<dt><b>7. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@7037@comp%2elang%2ec%2emoderated"><strong>ANNOUNCE: Date::Calc 4.1</strong></a><dd>10 Jun 98 - <b>comp.lang.c.moderated</b><br><a href="news:clcm-19980610-0029@plethora.net">&lt;clcm-19980610-0029@plethora.net&gt;</a><br>  <a href="mailto:sb@engelschall.com">sb@engelschall.com (Steffen Beyer)</a><P>
<dt><b>8. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@728@comp%2elang%2eperl%2eannounce"><strong>ANNOUNCE: Date::Calc 4.1</strong></a><dd>23 Jun 98 - <b>comp.lang.perl.announce</b><br><a href="news:6monvi$74t$1@news.neta.com">&lt;6monvi$74t$1@news.neta.com&gt;</a><br>  <a href="mailto:sb@engelschall.com">sb@engelschall.com (Steffen Beyer)</a><P>
<dt><b>9. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@729@comp%2elang%2eperl%2eannounce"><strong>ANNOUNCE: Data::Locations 4.1</strong></a><dd>23 Jun 98 - <b>comp.lang.perl.announce</b><br><a href="news:6monr0$70r$1@news.neta.com">&lt;6monr0$70r$1@news.neta.com&gt;</a><br>  <a href="mailto:sb@engelschall.com">sb@engelschall.com (Steffen Beyer)</a><P>
<dt><b>10. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@130246@comp%2elang%2eperl%2emisc"><strong>Manipulating @INC for module include path</strong></a><dd> 2 Jul 98 - <b>comp.lang.perl.misc</b><br><a href="news:359B4EE7.652296D1@kiel.netsruf.de">&lt;359B4EE7.652296D1@kiel.netsruf.de&gt;</a><br>  <a href="mailto:ulf.wendel@kiel.netsruf.de">Ulf Wendel &lt;ulf.wendel@kiel.netsruf.de&gt;</a><P>
<dt><b>11. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@131269@comp%2elang%2eperl%2emisc"><strong>Re: Manipulating @INC for module include path</strong></a><dd> 7 Jul 98 - <b>comp.lang.perl.misc</b><br><a href="news:35A2BAB1.446A40DA@infonline.net">&lt;35A2BAB1.446A40DA@infonline.net&gt;</a><br>  <a href="mailto:tonylabb@infonline.net">Tony Labbiento &lt;tonylabb@infonline.net&gt;</a><P>
<dt><b>12. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@138166@comp%2elang%2eperl%2emisc"><strong>Re: Where can I find the binary for perl at least 5.002 (better 5.005) for Windows NT4.0 ?</strong></a><dd>11 Aug 98 - <b>comp.lang.perl.misc</b><br><a href="news:8c7m0fr3a5.fsf@gadget.cscaper.com">&lt;8c7m0fr3a5.fsf@gadget.cscaper.com&gt;</a><br>  <a href="mailto:merlyn@stonehenge.com">Randal Schwartz &lt;merlyn@stonehenge.com&gt;</a><P>
<dt><b>13. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@153619@comp%2elang%2eperl%2emisc"><strong>ANNOUNCE: Bit::Vector 5.6</strong></a><dd> 2 Nov 98 - <b>comp.lang.perl.misc</b><br><a href="news:71le13$pns$1@bsdti6.sdm.de">&lt;71le13$pns$1@bsdti6.sdm.de&gt;</a><br>  <a href="mailto:sb@sdm.de">Steffen Beyer &lt;sb@sdm.de&gt;</a><P>
<dt><b>14. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@801@comp%2elang%2eperl%2eannounce"><strong>ANNOUNCE: Bit::Vector 5.6</strong></a><dd> 3 Nov 98 - <b>comp.lang.perl.announce</b><br><a href="news:71nb15$7vm$1@news.neta.com">&lt;71nb15$7vm$1@news.neta.com&gt;</a><br>  <a href="mailto:sb@sdm.de">Steffen Beyer &lt;sb@sdm.de&gt;</a><P>
<dt><b>15. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@34881@comp%2elang%2epython"><strong>Re: Painless Python Installer</strong></a><dd> 5 Nov 98 - <b>comp.lang.python</b><br><a href="news:71t037$2d1$1@news0-alterdial.uu.net">&lt;71t037$2d1$1@news0-alterdial.uu.net&gt;</a><br>  <a href="mailto:gward@thrak.cnri.reston.va.us">gward@thrak.cnri.reston.va.us (Greg Ward)</a><P>
<dt><b>16. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@35074@comp%2elang%2epython"><strong>Re: Painless Python Installer</strong></a><dd> 9 Nov 98 - <b>comp.lang.python</b><br><a href="news:3646B15E.F7732DDB@pop.vet.uu.nl">&lt;3646B15E.F7732DDB@pop.vet.uu.nl&gt;</a><br>  <a href="mailto:faassen@pop.vet.uu.nl">faassen &lt;faassen@pop.vet.uu.nl&gt;</a><P>
<dt><b>17. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@48830@comp%2eos%2elinux%2edevelopment%2eapps"><strong>Re: Creating Charts</strong></a><dd> 1 Sep 98 - <b>comp.os.linux.development.apps</b><br><a href="news:6sgsma$n7r@btmpjg.god.bel.alcatel.be">&lt;6sgsma$n7r@btmpjg.god.bel.alcatel.be&gt;</a><br>  <a href="mailto:verthezp@nemdev1">verthezp@nemdev1 (Peter Verthez)</a><P>
<dt><b>18. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@34830@comp%2elang%2epython"><strong>Installation progress (was: ODBC date problems)</strong></a><dd> 5 Nov 98 - <b>comp.lang.python</b><br><a href="news:71rtkm$4jb$1@Starbase.NeoSoft.COM">&lt;71rtkm$4jb$1@Starbase.NeoSoft.COM&gt;</a><br>  <a href="mailto:claird@Starbase.NeoSoft.COM">claird@Starbase.NeoSoft.COM (Cameron Laird)</a><P>
<dt><b>19. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@126693@comp%2elang%2eperl%2emisc"><strong>Re: Sending mail, can I set the $from ?</strong></a><dd>16 Jun 98 - <b>comp.lang.perl.misc</b><br><a href="news:35867929.5E440ECD@houston.Geco-Prakla.slb.com">&lt;35867929.5E440ECD@houston.Geco-Prakla.slb.com&gt;</a><br>  <a href="mailto:barnett@houston.Geco-Prakla.slb.com">Dave Barnett &lt;barnett@houston.Geco-Prakla.slb.com&gt;</a><P>
<dt><b>20. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@47986@comp%2edatabases%2einformix"><strong>Re: DBD:Informix test failure</strong></a><dd>26 Jul 98 - <b>comp.databases.informix</b><br><a href="news:35BBE0E3.5D36@earthlink.net">&lt;35BBE0E3.5D36@earthlink.net&gt;</a><br>  <a href="mailto:jleffler@earthlink.net">Jonathan Leffler &lt;jleffler@earthlink.net&gt;</a><P>
<dt><b>21. </b><a href="http://ww2.altavista.com/cgi-bin/news?msg@48048@comp%2edatabases%2einformix"><strong>Re: DBD:Informix test failure</strong></a><dd>26 Jul 98 - <b>comp.databases.informix</b><br><a href="news:35BBB49C.73C2@earthlink.net">&lt;35BBB49C.73C2@earthlink.net&gt;</a><br>  <a href="mailto:jleffler@earthlink.net">Jonathan Leffler &lt;jleffler@earthlink.net&gt;</a><P>
</dl>
<font size=-1><b>Result Pages: </b><b>1</b> &nbsp;<a href="/cgi-bin/query?pg=q&what=news&text=yes&q=%2bPerl+%2bCPAN&stq=30&c9k">2</a>
 &nbsp;<a href="/cgi-bin/query?pg=q&what=news&text=yes&q=%2bPerl+%2bCPAN&stq=60&c9k">3</a>
 &nbsp;<a href="/cgi-bin/query?pg=q&what=news&text=yes&q=%2bPerl+%2bCPAN&stq=90&c9k">4</a>
 &nbsp;<a href="/cgi-bin/query?pg=q&what=news&text=yes&q=%2bPerl+%2bCPAN&stq=120&c9k">5</a>
 &nbsp;<a href="/cgi-bin/query?pg=q&what=news&text=yes&q=%2bPerl+%2bCPAN&stq=150&c9k">6</a>
 &nbsp;<a href="/cgi-bin/query?pg=q&what=news&text=yes&q=%2bPerl+%2bCPAN&stq=180&c9k">7</a>
&nbsp; <a href="/cgi-bin/query?pg=q&what=news&text=yes&q=%2bPerl+%2bCPAN&stq=30&c9k">[<b>Next &gt;&gt;</b>]</a> <br>
</font><font size=-2>word count: CPAN: 7608; Perl: 171919<BR></font><P>
<br>
<font size=-2 face=arial, helvetica>
<a href="/cgi-bin/query?text">AltaVista Home</a> | 
<a href="/av/content/help.htm">Help</a> | 
<a href="/av/content/questions.htm">Feedback</a>
<br> 
Copyright | 
<a href="/av/content/disclaimer.htm">Disclaimer</a> | 
<a href="/av/content/privacy.htm">Privacy</a>
<br>
<a href="/av/content/about.htm">About AltaVista</a> | 
<a href="/cgi-bin/query?pg=tmpl&v=pref.html">Set your Preferences</a> | 
<a href="/av/content/addurl.htm">Add a Page</a> | 
<a href="/cgi-bin/query">Graphic Version</a>
</font>
</BODY></HTML>
