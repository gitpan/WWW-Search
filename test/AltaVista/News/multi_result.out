http://ww2.altavista.com/cgi-bin/news?msg@10743@de%2ecomp%2elang%2eperl
http://ww2.altavista.com/cgi-bin/news?msg@10757@de%2ecomp%2elang%2eperl
http://ww2.altavista.com/cgi-bin/news?msg@187281@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@187532@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@16582@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@161060@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@168147@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@168181@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@168324@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@18238@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@18240@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@18271@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@187670@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@188131@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@188249@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@188485@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@8606@comp%2edatabases%2exbase%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@175355@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@182771@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@188007@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@18404@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@894@comp%2elang%2eperl%2eannounce
http://ww2.altavista.com/cgi-bin/news?msg@1262@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@173481@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@185965@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@4709@comp%2eunix%2ebsd%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@240129@comp%2eos%2elinux%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@58570@comp%2einfosystems%2ewww%2eauthoring%2ecgi
http://ww2.altavista.com/cgi-bin/news?msg@20485@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@29992@comp%2eprotocols%2etcp%2dip
http://ww2.altavista.com/cgi-bin/news?msg@30020@comp%2eprotocols%2etcp%2dip
http://ww2.altavista.com/cgi-bin/news?msg@188023@comp%2eos%2elinux%2esetup
http://ww2.altavista.com/cgi-bin/news?msg@10144232@junk
http://ww2.altavista.com/cgi-bin/news?msg@123735@comp%2esys%2emac%2ecomm
http://ww2.altavista.com/cgi-bin/news?msg@186323@comp%2eos%2elinux%2esetup
http://ww2.altavista.com/cgi-bin/news?msg@187859@comp%2eos%2elinux%2esetup
http://ww2.altavista.com/cgi-bin/news?msg@187892@comp%2eos%2elinux%2esetup
http://ww2.altavista.com/cgi-bin/news?msg@188625@comp%2eos%2elinux%2esetup
http://ww2.altavista.com/cgi-bin/news?msg@188705@comp%2eos%2elinux%2esetup
http://ww2.altavista.com/cgi-bin/news?msg@54471@comp%2edatabases%2einformix
http://ww2.altavista.com/cgi-bin/news?msg@172969@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@184859@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@10678@comp%2elang%2eperl%2etk
http://ww2.altavista.com/cgi-bin/news?msg@20704@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@20732@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@207482@comp%2elang%2ec%2b%2b
http://ww2.altavista.com/cgi-bin/news?msg@156978@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@157115@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@1326@comp%2eunix%2esco%2eannounce
http://ww2.altavista.com/cgi-bin/news?msg@207794@comp%2elang%2ec%2b%2b
http://ww2.altavista.com/cgi-bin/news?msg@157244@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@207941@comp%2elang%2ec%2b%2b
http://ww2.altavista.com/cgi-bin/news?msg@157357@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@157508@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@157579@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@157828@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@157864@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@16169@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@16182@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@16184@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@158075@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@16234@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@158178@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@16246@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@158250@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@158299@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@16309@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@158773@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@158868@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@158954@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@439@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@444@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@445@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@448@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@159553@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@159577@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@55788@comp%2einfosystems%2ewww%2eauthoring%2ecgi
http://ww2.altavista.com/cgi-bin/news?msg@55792@comp%2einfosystems%2ewww%2eauthoring%2ecgi
http://ww2.altavista.com/cgi-bin/news?msg@159650@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@159736@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@16513@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@159914@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@466@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@36462@comp%2elang%2epython
http://ww2.altavista.com/cgi-bin/news?msg@160083@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@160094@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@6545@comp%2elang%2ejava%2esoftwaretools
http://ww2.altavista.com/cgi-bin/news?msg@160272@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@16601@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@16602@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@16612@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@16613@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@489@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@502@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@160781@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@16687@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@161074@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@56026@comp%2einfosystems%2ewww%2eauthoring%2ecgi
http://ww2.altavista.com/cgi-bin/news?msg@16753@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@75678@comp%2eos%2evms
http://ww2.altavista.com/cgi-bin/news?msg@75709@comp%2eos%2evms
http://ww2.altavista.com/cgi-bin/news?msg@161549@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@161669@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@838@comp%2elang%2eperl%2eannounce
http://ww2.altavista.com/cgi-bin/news?msg@75822@comp%2eos%2evms
http://ww2.altavista.com/cgi-bin/news?msg@162105@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@56164@comp%2einfosystems%2ewww%2eauthoring%2ecgi
http://ww2.altavista.com/cgi-bin/news?msg@162387@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@162411@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@409@alt%2eperl
http://ww2.altavista.com/cgi-bin/news?msg@162584@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@410@alt%2eperl
http://ww2.altavista.com/cgi-bin/news?msg@162624@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@162722@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@162777@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@17022@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@162987@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@163017@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@46666@comp%2einfosystems%2ewww%2eservers%2eunix
http://ww2.altavista.com/cgi-bin/news?msg@163368@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@17110@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@163459@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@163575@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@163585@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@198293@comp%2elang%2ejava%2eprogrammer
http://ww2.altavista.com/cgi-bin/news?msg@198438@comp%2elang%2ejava%2eprogrammer
http://ww2.altavista.com/cgi-bin/news?msg@17232@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@164386@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164388@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164674@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@37156@comp%2eunix%2eshell
http://ww2.altavista.com/cgi-bin/news?msg@164709@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164806@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164949@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@165125@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@17345@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@165539@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@165568@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@17425@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@165998@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@17458@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@166089@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@166221@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@163459@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@163575@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@198438@comp%2elang%2ejava%2eprogrammer
http://ww2.altavista.com/cgi-bin/news?msg@164062@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164085@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@17232@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@164418@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164439@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164446@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164596@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164806@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164949@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@165539@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@24479@comp%2eobject%2ecorba
http://ww2.altavista.com/cgi-bin/news?msg@165998@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@166287@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@166345@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@826@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@828@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@835@comp%2elang%2eperl%2emoderated
